2024-03-02 12:50:48,523 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 12:51:06,842 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 12:51:09,584 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 12:53:09,253 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 12:53:14,075 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 12:53:14,181 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 12:53:23,324 ERROR NewReminderASP.WebUI.Areas.ContactsArea.Controllers.PhoneController - An unhandled exception occurred
System.NullReferenceException: —сылка на объект не указывает на экземпл€р объекта.
   в ASP._Page_Areas_ContactsArea_Views_Phone_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\ContactsArea\Views\Phone\Create.cshtml:строка 72
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 12:53:27,599 ERROR NewReminderASP.WebUI.Areas.ContactsArea.Controllers.PhoneController - An unhandled exception occurred
System.NullReferenceException: —сылка на объект не указывает на экземпл€р объекта.
   в ASP._Page_Areas_ContactsArea_Views_Phone_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\ContactsArea\Views\Phone\Create.cshtml:строка 72
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 12:53:49,196 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 12:53:55,583 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 12:53:55,770 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 12:58:35,664 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 12:58:42,144 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 12:58:43,220 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:04:37,699 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:04:43,796 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:04:43,998 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:05:26,876 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPhoneService.AddUserPhoneRegister(UserPhoneDto userPhone)
   в NewReminderASP.Data.ServiceReference1.PhoneServiceClient.AddUserPhoneRegister(UserPhoneDto userPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 1080
   в NewReminderASP.Data.Client.PhoneClient.AddUserPhoneRegister(UserPhone userPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PhoneClient.cs:строка 160
2024-03-02 13:13:23,130 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:13:31,228 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:13:31,930 WARN NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Authentication failed for user with login: work
2024-03-02 13:13:34,379 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:13:34,489 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:17:16,572 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:17:23,234 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:17:23,932 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:18:27,433 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:18:31,900 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:18:31,997 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:18:52,230 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:18:55,970 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:18:56,065 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:19:15,311 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:19:20,069 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:19:20,169 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:19:46,084 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:19:50,554 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:19:50,651 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:20:09,019 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:20:13,347 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:20:13,478 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:23:25,811 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:23:30,499 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:23:30,607 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:28:20,347 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:28:28,388 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:28:28,548 WARN NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Authentication failed for user with login: work
2024-03-02 13:28:30,849 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:28:30,947 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:28:56,552 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPhoneService.UpdateUserPhone(UserPhoneDto updatedUserPhone)
   в NewReminderASP.Data.ServiceReference1.PhoneServiceClient.UpdateUserPhone(UserPhoneDto updatedUserPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 1064
   в NewReminderASP.Data.Client.PhoneClient.UpdateUserPhone(UserPhone updatedUserPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PhoneClient.cs:строка 100
2024-03-02 13:32:02,812 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:32:07,521 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:32:07,621 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:32:23,650 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPhoneService.UpdateUserPhone(UserPhoneDto updatedUserPhone)
   в NewReminderASP.Data.ServiceReference1.PhoneServiceClient.UpdateUserPhone(UserPhoneDto updatedUserPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 1064
   в NewReminderASP.Data.Client.PhoneClient.UpdateUserPhone(UserPhone updatedUserPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PhoneClient.cs:строка 100
2024-03-02 13:40:56,343 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:41:03,462 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 13:41:04,166 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 13:42:29,947 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 13:42:38,923 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: pa
2024-03-02 13:42:39,054 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:22:04,915 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:22:11,980 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:22:13,974 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:23:07,466 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPhoneService.AddPhoneType(PhoneTypeDto eventPhoneType)
   в NewReminderASP.Data.ServiceReference1.PhoneServiceClient.AddPhoneType(PhoneTypeDto eventPhoneType) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 1120
   в NewReminderASP.Data.Client.PhoneClient.AddPhoneType(PhoneType eventPhoneType) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PhoneClient.cs:строка 310
2024-03-02 14:24:02,883 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:24:11,086 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:24:11,822 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:25:45,989 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:26:19,203 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:26:19,291 WARN NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Authentication failed for user with login: work
2024-03-02 14:26:22,418 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:26:22,517 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:27:09,717 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:27:14,502 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:27:14,600 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:28:19,150 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:28:23,706 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:28:23,804 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:28:35,832 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 128
2024-03-02 14:28:35,836 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 146
   в NewReminderASP.Data.Repository.PersonalInformationRepository.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\PersonalInformationRepository.cs:строка 34
   в NewReminderASP.Core.Provider.PersonalInformationProvider.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\PersonalInformationProvider.cs:строка 35
   в NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController.Create(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Controllers\InfoController.cs:строка 83
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:33:27,674 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:33:35,296 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:33:36,012 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:34:22,663 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 128
2024-03-02 14:34:22,667 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 146
   в NewReminderASP.Data.Repository.PersonalInformationRepository.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\PersonalInformationRepository.cs:строка 34
   в NewReminderASP.Core.Provider.PersonalInformationProvider.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\PersonalInformationProvider.cs:строка 35
   в NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController.Create(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Controllers\InfoController.cs:строка 83
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:34:30,936 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 128
2024-03-02 14:34:30,936 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 146
   в NewReminderASP.Data.Repository.PersonalInformationRepository.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\PersonalInformationRepository.cs:строка 34
   в NewReminderASP.Core.Provider.PersonalInformationProvider.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\PersonalInformationProvider.cs:строка 35
   в NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController.Create(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Controllers\InfoController.cs:строка 83
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:37:41,757 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:37:48,270 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:37:48,982 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:38:44,063 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:39:32,943 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:39:40,521 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:39:46,772 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:39:47,469 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:39:56,571 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:40:02,174 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:40:11,027 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:40:17,738 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:40:31,473 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:40:38,348 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:40:38,446 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:40:50,005 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:41:05,409 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:41:13,810 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ArgumentNullException: «начение не может быть неопределенным.
»м€ параметра: items
   в System.Web.Mvc.MultiSelectList..ctor(IEnumerable items, String dataValueField, String dataTextField, String dataGroupField, IEnumerable selectedValues, IEnumerable disabledValues, IEnumerable disabledGroups)
   в System.Web.Mvc.SelectList..ctor(IEnumerable items, String dataValueField, String dataTextField)
   в ASP._Page_Areas_PersonInfoArea_Views_Info_Create_cshtml.Execute() в e:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Views\Info\Create.cshtml:строка 57
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy()
   в System.Web.Mvc.WebViewPage.ExecutePageHierarchy()
   в System.Web.WebPages.StartPage.ExecutePageHierarchy()
   в System.Web.WebPages.WebPageBase.ExecutePageHierarchy(WebPageContext pageContext, TextWriter writer, WebPageRenderingBase startPage)
   в System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilterRecursive(IList`1 filters, Int32 filterIndex, ResultExecutingContext preContext, ControllerContext controllerContext, ActionResult actionResult)
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionResultWithFilters(ControllerContext controllerContext, IList`1 filters, ActionResult actionResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:41:40,121 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:41:46,788 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:41:46,969 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:42:30,005 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 128
2024-03-02 14:43:18,332 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:43:24,098 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:43:24,778 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:47:19,458 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:47:25,901 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:47:26,090 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:48:55,465 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:49:01,819 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:49:02,605 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:49:48,882 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 128
2024-03-02 14:51:31,206 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:51:37,793 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:51:37,981 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:52:07,509 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 128
2024-03-02 14:52:07,513 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 146
   в NewReminderASP.Data.Repository.PersonalInformationRepository.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\PersonalInformationRepository.cs:строка 34
   в NewReminderASP.Core.Provider.PersonalInformationProvider.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\PersonalInformationProvider.cs:строка 35
   в NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController.Create(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Controllers\InfoController.cs:строка 84
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 14:54:55,195 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:55:01,719 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:55:01,897 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 14:58:26,594 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 14:58:31,113 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 14:58:31,218 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:03:29,741 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:03:34,341 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:03:34,454 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:12:17,970 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:13:49,009 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:13:56,303 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:13:57,011 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:15:16,034 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:15:30,540 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:15:46,910 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:15:47,046 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:16:16,879 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:16:23,566 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:16:24,269 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:16:35,234 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:16:42,985 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:16:43,086 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:17:09,643 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:17:23,261 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:17:54,194 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:18:00,248 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:18:00,949 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:18:45,884 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:18:52,990 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:18:53,760 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:19:09,987 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:20:01,146 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:20:07,230 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:20:07,938 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:20:31,488 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 128
2024-03-02 15:20:31,491 ERROR NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPersonalInfoService.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo)
   в NewReminderASP.Data.ServiceReference1.PersonalInfoServiceClient.AddPersonalInfo(String userLogin, PersonalInfoDto personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 620
   в NewReminderASP.Data.Client.PersonalInformationClient.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PersonalInformationClient.cs:строка 146
   в NewReminderASP.Data.Repository.PersonalInformationRepository.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\PersonalInformationRepository.cs:строка 34
   в NewReminderASP.Core.Provider.PersonalInformationProvider.AddPersonalInfo(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\PersonalInformationProvider.cs:строка 35
   в NewReminderASP.WebUI.Areas.PersonInfoArea.Controllers.InfoController.Create(String userLogin, PersonalInfo personalInfo) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\PersonInfoArea\Controllers\InfoController.cs:строка 84
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 15:22:03,473 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:22:41,228 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:22:48,917 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:22:49,664 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:23:08,490 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IPhoneService.AddUserPhoneRegister(UserPhoneDto userPhone)
   в NewReminderASP.Data.ServiceReference1.PhoneServiceClient.AddUserPhoneRegister(UserPhoneDto userPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 1080
   в NewReminderASP.Data.Client.PhoneClient.AddUserPhoneRegister(UserPhone userPhone) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\PhoneClient.cs:строка 160
2024-03-02 15:23:47,026 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:23:53,807 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:23:53,903 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:24:34,488 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:24:38,122 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:24:38,226 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:25:12,711 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:25:29,492 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:25:38,058 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:25:38,197 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:30:32,647 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:34:47,456 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:36:19,599 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:37:11,009 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:38:26,990 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:38:37,531 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:38:37,669 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:40:33,671 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:40:39,784 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: pa
2024-03-02 15:40:39,903 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:42:13,324 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:42:44,565 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:42:53,618 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:42:53,740 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:45:25,314 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:45:37,180 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:45:37,283 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:49:45,427 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:49:52,052 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:49:52,115 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:49:52,753 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:49:52,772 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IUserService.AssignRolesToUser(UserDto user, String[] roles)
   в NewReminderASP.Data.ServiceReference1.UserServiceClient.AssignRolesToUser(UserDto user, String[] roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 283
   в UserClient.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\UserClient.cs:строка 29
2024-03-02 15:49:52,775 ERROR NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IUserService.AssignRolesToUser(UserDto user, String[] roles)
   в NewReminderASP.Data.ServiceReference1.UserServiceClient.AssignRolesToUser(UserDto user, String[] roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 283
   в UserClient.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\UserClient.cs:строка 38
   в NewReminderASP.Data.Repository.UserRepository.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\UserRepository.cs:строка 102
   в NewReminderASP.Core.Provider.UserProvider.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\UserProvider.cs:строка 124
   в NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController.Login(String login, String password, String returnUrl) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\LoginArea\Controllers\LoginController.cs:строка 47
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 15:49:58,307 ERROR NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - An unhandled exception occurred
System.Web.Mvc.HttpAntiForgeryException (0x80004005): The provided anti-forgery token was meant for user "", but the current user is "work".
   в System.Web.Helpers.AntiXsrf.TokenValidator.ValidateTokens(HttpContextBase httpContext, IIdentity identity, AntiForgeryToken sessionToken, AntiForgeryToken fieldToken)
   в System.Web.Helpers.AntiXsrf.AntiForgeryWorker.Validate(HttpContextBase httpContext)
   в System.Web.Mvc.ControllerActionInvoker.InvokeAuthorizationFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__0(AsyncCallback asyncCallback, Object asyncState)
2024-03-02 15:50:05,905 ERROR NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - An unhandled exception occurred
System.Web.Mvc.HttpAntiForgeryException (0x80004005): The provided anti-forgery token was meant for user "", but the current user is "work".
   в System.Web.Helpers.AntiXsrf.TokenValidator.ValidateTokens(HttpContextBase httpContext, IIdentity identity, AntiForgeryToken sessionToken, AntiForgeryToken fieldToken)
   в System.Web.Helpers.AntiXsrf.AntiForgeryWorker.Validate(HttpContextBase httpContext)
   в System.Web.Mvc.ControllerActionInvoker.InvokeAuthorizationFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__0(AsyncCallback asyncCallback, Object asyncState)
2024-03-02 15:50:12,001 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:50:16,961 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: pa
2024-03-02 15:50:17,087 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:50:35,553 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 202
2024-03-02 15:50:35,553 ERROR NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 220
   в NewReminderASP.Data.Repository.AddressRepository.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\AddressRepository.cs:строка 44
   в NewReminderASP.Core.Provider.AddressProvider.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\AddressProvider.cs:строка 43
   в NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController.Create(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\AddressArea\Controllers\AddressController.cs:строка 83
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 15:50:45,234 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 202
2024-03-02 15:50:45,235 ERROR NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 220
   в NewReminderASP.Data.Repository.AddressRepository.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\AddressRepository.cs:строка 44
   в NewReminderASP.Core.Provider.AddressProvider.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\AddressProvider.cs:строка 43
   в NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController.Create(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\AddressArea\Controllers\AddressController.cs:строка 83
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 15:51:42,384 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 202
2024-03-02 15:51:42,384 ERROR NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 220
   в NewReminderASP.Data.Repository.AddressRepository.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\AddressRepository.cs:строка 44
   в NewReminderASP.Core.Provider.AddressProvider.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\AddressProvider.cs:строка 43
   в NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController.Create(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\AddressArea\Controllers\AddressController.cs:строка 83
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 15:51:49,309 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 202
2024-03-02 15:51:49,309 ERROR NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IAddressService.AddAddressRegister(AddressDto address)
   в NewReminderASP.Data.ServiceReference1.AddressServiceClient.AddAddressRegister(AddressDto address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 517
   в NewReminderASP.Data.Client.AddressClient.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\AddressClient.cs:строка 220
   в NewReminderASP.Data.Repository.AddressRepository.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\AddressRepository.cs:строка 44
   в NewReminderASP.Core.Provider.AddressProvider.AddAddressRegister(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\AddressProvider.cs:строка 43
   в NewReminderASP.WebUI.Areas.AddressArea.Controllers.AddressController.Create(Address address) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\AddressArea\Controllers\AddressController.cs:строка 83
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 15:52:34,678 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:52:39,015 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:52:39,117 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:53:30,398 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:53:36,712 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:53:36,740 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 15:53:37,487 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 15:53:37,513 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IUserService.AssignRolesToUser(UserDto user, String[] roles)
   в NewReminderASP.Data.ServiceReference1.UserServiceClient.AssignRolesToUser(UserDto user, String[] roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 283
   в UserClient.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\UserClient.cs:строка 29
2024-03-02 15:53:37,516 ERROR NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - An unhandled exception occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IUserService.AssignRolesToUser(UserDto user, String[] roles)
   в NewReminderASP.Data.ServiceReference1.UserServiceClient.AssignRolesToUser(UserDto user, String[] roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 283
   в UserClient.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\UserClient.cs:строка 38
   в NewReminderASP.Data.Repository.UserRepository.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Repository\UserRepository.cs:строка 102
   в NewReminderASP.Core.Provider.UserProvider.AssignRolesToUser(User user, List`1 roles) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Core\Provider\UserProvider.cs:строка 124
   в NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController.Login(String login, String password, String returnUrl) в E:\CSharp\NewReminderASP.Core\NewReminderASP.WebUI\Areas\LoginArea\Controllers\LoginController.cs:строка 47
   в lambda_method(Closure , ControllerBase , Object[] )
   в System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c.<BeginInvokeSynchronousActionMethod>b__9_0(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
   в System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_0.<InvokeActionMethodFilterAsynchronouslyRecursive>b__0()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass11_2.<InvokeActionMethodFilterAsynchronouslyRecursive>b__2()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_5.<BeginInvokeAction>b__4()
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__1(IAsyncResult asyncResult)
2024-03-02 15:53:46,508 ERROR NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - An unhandled exception occurred
System.Web.Mvc.HttpAntiForgeryException (0x80004005): The provided anti-forgery token was meant for user "", but the current user is "work".
   в System.Web.Helpers.AntiXsrf.TokenValidator.ValidateTokens(HttpContextBase httpContext, IIdentity identity, AntiForgeryToken sessionToken, AntiForgeryToken fieldToken)
   в System.Web.Helpers.AntiXsrf.AntiForgeryWorker.Validate(HttpContextBase httpContext)
   в System.Web.Mvc.ControllerActionInvoker.InvokeAuthorizationFilters(ControllerContext controllerContext, IList`1 filters, ActionDescriptor actionDescriptor)
   в System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass3_0.<BeginInvokeAction>b__0(AsyncCallback asyncCallback, Object asyncState)
2024-03-02 15:53:57,519 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 15:54:01,848 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: pa
2024-03-02 15:54:01,953 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 16:04:26,583 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 16:04:33,162 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 16:04:33,852 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 16:09:49,726 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 16:09:56,151 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 16:09:56,827 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 16:13:31,067 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 16:13:40,595 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 16:13:41,313 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 16:15:47,540 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:10:12,394 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:10:18,102 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:10:20,148 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:12:06,178 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:12:10,621 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:12:10,726 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:13:24,088 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:13:29,528 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:13:29,630 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:13:56,513 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IEventService.AddPEventType(EventTypeDto eventType)
   в NewReminderASP.Data.ServiceReference1.EventServiceClient.AddPEventType(EventTypeDto eventType) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 933
   в NewReminderASP.Data.Client.EventClient.AddPEventType(EventType eventType) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\EventClient.cs:строка 581
2024-03-02 17:13:56,514 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IEventService.AddPEventType(EventTypeDto eventType)
   в NewReminderASP.Data.ServiceReference1.EventServiceClient.AddPEventType(EventTypeDto eventType) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 933
   в NewReminderASP.Data.Client.EventClient.AddPEventType(EventType eventType) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\EventClient.cs:строка 581
2024-03-02 17:16:32,730 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:16:39,064 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:16:39,893 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:30:12,051 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:30:18,958 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:30:19,681 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:38:48,401 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:38:55,359 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:38:55,568 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:41:43,530 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:41:48,528 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:41:48,637 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:42:42,134 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:42:45,787 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:42:45,887 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:44:03,294 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:44:07,073 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:44:07,190 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:52:44,093 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:52:49,129 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: Work
2024-03-02 17:52:49,249 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:56:02,158 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:56:06,529 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:56:06,650 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:56:54,173 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:56:58,368 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:56:58,477 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:57:33,033 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:57:40,024 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:57:40,775 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 17:59:00,823 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 17:59:06,952 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 17:59:07,072 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 19:45:06,827 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 19:45:15,186 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 19:45:17,989 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 19:46:10,161 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IEventService.AddEvent(EventDto events)
   в NewReminderASP.Data.ServiceReference1.EventServiceClient.AddEvent(EventDto events) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 813
   в NewReminderASP.Data.Client.EventClient.AddEvent(Event events) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\EventClient.cs:строка 134
2024-03-02 19:46:29,757 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 19:46:33,993 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 19:46:34,092 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 19:49:32,799 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 19:49:38,966 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 19:49:39,075 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 19:52:08,315 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 19:52:12,935 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 19:52:13,064 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 19:56:15,606 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 19:56:30,134 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 19:56:31,194 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 20:06:28,484 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 20:06:35,476 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 20:06:35,676 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 20:10:13,736 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 20:10:18,605 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 20:10:18,706 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 20:14:43,970 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 20:14:51,285 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 20:14:52,016 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 20:15:13,669 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IEventService.AddEventDetail(EventDetailDto eventDetail)
   в NewReminderASP.Data.ServiceReference1.EventServiceClient.AddEventDetail(EventDetailDto eventDetail) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 853
   в NewReminderASP.Data.Client.EventClient.AddEventDetail(EventDetail eventDetail) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\EventClient.cs:строка 288
2024-03-02 20:16:21,374 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Rendering AuthenticateUser view
2024-03-02 20:16:27,778 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - Attempting user authentication for login: work
2024-03-02 20:16:27,898 INFO NewReminderASP.WebUI.Areas.LoginArea.Controllers.LoginController - User authenticated successfully. Assigned roles: Admin
2024-03-02 20:17:11,187 ERROR ErrorLogger - An error occurred
System.ServiceModel.FaultException: —ервер не смог обработать запрос в св€зи с внутренней ошибкой.  ƒл€ получени€ дополнительных сведений об ошибке либо включите IncludeExceptionDetailInFaults (с помощью атрибута ServiceBehaviorAttribute или через поведение < serviceDebug >) на сервере дл€ того чтобы выслать информацию об исключении назад клиенту, или включить трассировку согласно инструкции Microsoft .NET Framework SDK и проверить записи журнала трассировки клиента.

Server stack trace: 
   в System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   в System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в NewReminderASP.Data.ServiceReference1.IEventService.AddEventDetail(EventDetailDto eventDetail)
   в NewReminderASP.Data.ServiceReference1.EventServiceClient.AddEventDetail(EventDetailDto eventDetail) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Connected Services\ServiceReference1\Reference.cs:строка 853
   в NewReminderASP.Data.Client.EventClient.AddEventDetail(EventDetail eventDetail) в E:\CSharp\NewReminderASP.Core\NewReminderASP.Data\Client\EventClient.cs:строка 288
